1.0/docs/devguide/registering-elements.md

```diff
@@ -177,17 +177,75 @@ all imports in the document have finished loading.
 ## Lifecycle callbacks {#lifecycle-callbacks}
 
 Polymer's Base prototype implements the standard Custom Element lifecycle
-callbacks to perform tasks necessary for Polymer's built-in features.  The hooks
-in turn call shorter-named lifecycle methods on your prototype.
-
-- `created` instead of `createdCallback`
-- `attached` instead of `attachedCallback`
-- `detached` instead of `detachedCallback`
-- `attributeChanged` instead of `attributeChangedCallback`
+callbacks to perform tasks necessary for Polymer's built-in features.
+{{site.project_title}} in turn calls shorter-named lifecycle methods on your
+prototype.
 
 Polymer adds an extra callback, `ready`, which is invoked when Polymer has
 finished creating and initializing the element's local DOM.
 
+<table>
+  <tr>
+    <th>Callback</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><code>created</code></td>
+    <td>Called when the element has been created, but before property values are
+       set and local DOM is initialized.
+      <p>Use for one-time set-up before property values are set.
+      </p>
+      <p>Use instead of <code>createdCallback</code>.
+      </p>
+    </td>
+  </tr>
+  <tr>
+    <td><code>ready</code></td>
+    <td>Called after property values are set and local DOM is initialized.
+      <p>Use for one-time configuration of your component after local
+        DOM is initialized. (For configuration based on property values, it
+        may be preferable to use an <a href="properties.html#multi-property-observers">observer</a>.)
+      </p>
+    </td>
+  </tr>
+  <tr>
+    <td><code>attached</code></td>
```

Comment:
For elements that are written explicitly in the template (/ initial state) of their container, I assume that attached follows ready, but this could be good to make explicit.